sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -383,7 +383,7 @@ def run_pipeline(self, pipeline, options):
     use_fnapi = apiclient._use_fnapi(options)
     from apache_beam.portability.api import beam_runner_api_pb2
     default_container_image = (
```

Comment:
Maybe do not call this variable "default"_container_image any more, since it not really just default.

Alternatively, we can remove the variable and change L390 to read `container_image=apiclient.get_container_image_from_options(options)`

Reply:
I think it still is considered the "default" for the context of this pipeline.